pkg/projects/dir.targets

```diff
@@ -89,4 +89,20 @@
               ItemName="Dependency" />
     </MSBuild>
   </Target>
+
+  <!-- Remove the linux distro specific dependencies so the portable linux RID is used by default. -->
+  <Target Name="RemoveLinuxDistroDependencies"
+          Condition="'@(RuntimeDependency)' != ''"
+          AfterTargets="DetermineRuntimeDependencies"
+          Outputs="%(LinuxDistroRIDNonDependency.Identity)">
+    <PropertyGroup>
+      <CurrentLinuxDistroRID>%(LinuxDistroRIDNonDependency.Identity)</CurrentLinuxDistroRID>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_RuntimeDependencyToRemove Include="@(RuntimeDependency)"
+                                  Condition="'%(RuntimeDependency.TargetRuntime)' == '$(CurrentLinuxDistroRID)'" />
```

Comment:
Something along the lines of this should allow you to remove the batching from target.
`Condition="'%(RuntimeDependency.TargetRuntime)' == '%(CurrentLinuxDistroRID.Identity)'"`